.circleci/config.yml

```diff
@@ -666,48 +666,21 @@ jobs:
           command: |
             cd << parameters.repo >>
             audius_loggly_token_b64=$(echo ${audius_loggly_token} | base64)
-            docker build -t audius/<< parameters.repo >>:$IMAGE_TAG -t audius/<< parameters.repo>>:$(git rev-parse HEAD) --build-arg git_sha=$(git rev-parse HEAD) --build-arg audius_loggly_disable=$audius_loggly_disable --build-arg audius_loggly_token=$audius_loggly_token --build-arg audius_loggly_tags=$audius_loggly_tags --build-arg BUILD_NUM=$CIRCLE_BUILD_NUM --build-arg audius_loggly_token=$audius_loggly_token_b64 .
+            # two -t's are needed to ensure that the image is tagged with IMAGE_TAG and git SHA
+            docker build \
+              -t audius/<< parameters.repo >>:$IMAGE_TAG -t audius/<< parameters.repo>>:$(git rev-parse HEAD) \
```

Comment:
I'd break this into two lines where we have one line per parameter.

It makes it easier to read and doesn't hide two options in one line.

Reply:
fixed